src/methods.ts

```diff
@@ -110,6 +113,48 @@ class JsonRpcMethods {
             })
             .catch((e) => nodeError(e, callback));
     }
+
+    getInstanceInfo(
+        blockHash: string,
+        address: string,
```

Comment:
I think it would be nicer to actually get the structured values:
```
{
    "index": 0,
    "subindex": 0
}
```
and then do `JSON.stringify()` when creating the `GetAddressInfoRequest` object. I am a bit surprised that the gRPC request just takes a JSON blob :frowning: